### PR TITLE
add sep6 docs for wallet-sdk

### PR DESCRIPTION
**Pull request by acharb**
[ticket](https://stellarorg.atlassian.net/browse/WAL-1065?atlOrigin=eyJpIjoiM2IxODM5MDdjZTNhNDczOGFhZmM3NTllMmQwNjczOTgiLCJwIjoiaiJ9)

Adding sep6 and sep12 docs to the wallet sdk docs

### CODE REVIEW

File: docs/building-apps/wallet/sep6.mdx [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  params: {
+    asset_code,
+    account,
+  },
+});
+```
+
+</CodeExample>
+
+There are several kinds of responses, depending on if the anchor needs more information. All deposits and withdrawals will have these same response types:
```
Comment (Ifropc) [inline docs/building-apps/wallet/sep6.mdx:67]: Can you add code example of how to handle different response?

File: docs/building-apps/wallet/sep6.mdx [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+The response follows the same types as all the deposits and withdrawals for SEP-6.
+
+## Getting Transaction Info
+
+On the typical flow, the wallet would get transaction data to notify users about status updates. This is done via the [SEP-6] `GET /transaction` and `GET /transactions` endpoint.
+
+### Tracking Transaction
+
+Let's look into how to use the sdk to track transaction status changes. We will use the `Watcher` class for this purpose. First, let's initialize it and start tracking a transaction.
```
Comment (Ifropc) [inline docs/building-apps/wallet/sep6.mdx:202]: For sep-24 watcher also has refresh, is it not the case for SEP-24?

File: docs/building-apps/wallet/sep6.mdx (hunk 1 above)
Comment (acharb) [reply docs/building-apps/wallet/sep6.mdx:67]: good idea, hows this https://github.com/stellar/stellar-docs/pull/283/commits/1fb446fbafba1b90bbeba3cbb1d6f2c74685cff0

for the other 403 response I don't think it's worth adding a code example, there's not a standard way to handle

File: docs/building-apps/wallet/sep6.mdx (hunk 2 above)
Comment (acharb) [reply docs/building-apps/wallet/sep6.mdx:239]: good catch 👍 https://github.com/stellar/stellar-docs/pull/283/commits/70257245562bcba23ed19eef415cf106495077f3

**Review by Ifropc — APPROVED** [review]

**Pull request merged**